docs/setup/vscode-web.md

```diff
@@ -79,11 +79,16 @@ We've explored a couple of URLs already (`vscode.dev/github` and `vscode.dev/azu
 | MakeCode | `/edu/makecode` | [MakeCode docs](https://arcade.makecode.com/vscode) |
 | VS Code for Education | `/edu` | [VS Code for Education landing page](https://vscodeedu.com/) |
 | Azure Machine Learning (AML) | `/+ms-toolsai.vscode-ai-remote-web` | [AML docs](https://learn.microsoft.com/azure/machine-learning/how-to-launch-vs-code-remote?view=azureml-api-2&tabs=vscode-web) |
+| Azure | `/azure` | [AML docs](https://insiders.vscode.dev/azure) |
 
 Please note that some URLs must be entered in a specific way (for example, `vscode.dev/editor/liveshare` requires an active Live Share session). Please review each service's documentation for specific access and usage information.
 
 There's more information on some of these URLs below.
 
+## VS Code for the Web - Azure
```

Comment:
Move this section higher up, just below **Azure Repos**. It breaks the follow-on sections and fits better alongside the other Azure section.

Suggest renaming it to "Manage Azure applications"

Reply:
@meeraharidasa As mentioned above, please move this entire section above the "More custom URLs" and change the section name to "Manage Azure applications".
Then in the table with custom URLs, the URL should point to this section instead to make it consistent with how it's done for Azure Repos.

Reply:
@meeraharidasa Given that @mistymadonna is working on a separate page for the `/azure` endpoint, I suggest removing this detailed section and link to that new page from the table above.

Reply:
@mistymadonna can you help me with this update that @ntrogh suggested?